APE0.rst

```diff
@@ -0,0 +1,360 @@
+is a two-week comment period, and two-thirds of the active Voting Members have
+to approve the removal. In the period between removal/resignation and new
+appointment, the Coordination Committee will temporarily take over the
+responsibilities of the Ombudsperson, should the need arise.
+
+The Voting Members
+------------------
+
+Role
+^^^^
+The Voting Members are the group of trusted individuals who operate the Astropy
+Project on behalf of the community. They have authority over the Astropy
+Project’s technical resources, including the Astropy Project website itself, the
+Astropy GitHub organization and repositories, the issue tracker, and all Astropy
+communication channels. In practice, much of this authority is passed to the
+Coordination Committee via the voting processes described in this Charter.
+
+They also assume many roles required to achieve the Project's goals, especially
+those that require a high level of trust. Collectively, they make decisions that
+shape the future of the Project.
+
+Voting Members are expected to act as role models for the community and
+custodians of the Project, on behalf of the community and all those who rely on
+Astropy. They will act as representatives of the Astropy Project, where
+necessary, including in online discussions or at official Astropy events.
+
+Powers
+^^^^^^
+Voting Members may participate in formal votes on:
+* adding new Voting Members,
+* removing Voting Members,
+* electing Coordination Committee members,
+* removing Coordination Committee members,
+* appointment or removal of the Ombudsperson,
+* changes to this Charter,
+* other matters for which the Coordination Committee believes a vote is
+  appropriate.
+
+The mechanism, timeline, and criteria for a decisive vote are specified in the
+respective sections of this document in the first six cases and by the
+Coordination Committee in the last case.
+
+Membership
+^^^^^^^^^^
+Voting Members of the Astropy Project must demonstrate:
+* a good grasp of the philosophy of the Astropy Project,
+* a solid track record of being constructive and helpful,
+* significant contributions to the Project's goals, in any form,
+* a willingness to dedicate time to improving the Project,
+* a willingness to recruit, train, and nominate new team members.
+
+Voting Membership acknowledges sustained and valuable efforts that align well
+with the philosophy and the goals of the Astropy Project.
+
+Initial Membership
+^^^^^^^^^^^^^^^^^^
+Anyone who satisfies any of the following criteria at the date of acceptance of
+this APE will be invited to be in the first group of Voting Members:
+
+* has a named role in the project and has participated in an Astropy
+  coordination meeting in the last two years,
+* has a named role in the project and has regularly participated in Astropy
+  telecons (e.g., co-working telecon, spectroscopy sprint, Astropy Learn
+  telecon) in the last two years,
```

Comment:
It says "e.g.", so as it stands, the list is not exhaustive. However, how do we define the list in the beginning? For that, a complete list would be needed. I suggest that we make a list of people that would be initial voting members based on these criteria. We email that list to the community and say (in nicer wording) "Did we forget you? If so, we're sorry. Please leave comment / open PR here." From there, the current Coco can approve or deny, using the criteria here. Yes, that's in some way the Coco choosing the electorate, which is not a good thing in general, but I can't come up with a better way that's manageable. It's a chicken-and-egg problem. We (the most active members) should also monitor that list. Self-nominations for membership will just get the loudest, not the most involved to sign up. Thus, we should look at that list and say: "Oh, person A from telecon XYZ singed up. Looks like we forgot to add the people working on XYZ to the initial list. Who else participated in XYZ and should be added, even if they were not on the initial list and did not self-nominate for membership?"

Reply:
To clarify the idea from the WG was that we initialize the list using a not-perfect, but probably good enough heuristic, which is what this list is supposed to be.  Then there would be around where those people vote in other voting members.  The WG basically spent many hours trying to come up with a metric that would work without reaching consensus (or, I believe, even majority-vote agreement), so something more flexible like that was thought to be the safer way to go. 

So that would effectively be the same as @hamogu's proposal here, but with a body broader than the CoCo able to make that determination (it would include the CoCo though, since of course all current members meet these criteria).

At some point someone (@Cadair ?  Or maybe @astrofrog? Or maybe it was @bsipocz?) tabulated who would be in the initial set based on these criteria, and a few alternative versions, and it was always the same people plus-or-minus one or two (who would presumably get in on the first voting round anyway).  

(Note this connects to https://github.com/astropy/astropy-APEs/pull/61#issuecomment-704723190)

(But I agree the dev telecon would be included on this list!)

Reply:
You might recall an email in which I asked for a schedule reshuffle about this topic but only met with passive aggressive and exclusionary comments from two and the ask got ignored, while similar asks were always honoured in all previous meetings. So no, it wasn't me. 

And yes, it does matter what is in this initial requirement list as it puts down a baseline of what is a minimal level of involvement required.

Reply:
> You might recall an email  

I do not recall that email, and I'm not confident that everyone else in the WG interprets the events the same way, but I'm sorry that happened to you regardless. I don't think that's especially germain to this particular thread, though.

> And yes, it does matter what is in this initial requirement list as it puts down a baseline of what is a minimal level of involvement required.

That is not the intent as I recall the discussion in the WG - it is meant to set an initial list and then that initial list of members can accept others based on the wording in the "membership" section that's just above this one. (@Cadair may correct me if I misunderstood as I believe it was him who proposed this?)

Reply:
I think it's very important that the initial bootstrapping of the voting members doesn't set an ongoing precedent, and the wording should be strengthened if it is being interpreted that way. Future decisions on membership should be made solely on the grounds of the membership section.

It's obviously very important that we get this right, but it is a difficult problem to solve, balancing fairness and simplicity. I think the most important criteria for judging this section is **does everyone selected by this set of criteria meet the criteria for membership above**. If there is anyone being selected by these criteria that doesn't then they need updating, however, if they don't then they are serving their purpose to get us over the bootstrapping issue, and from acceptance of this document onwards the only thing that should matter is the membership criteria.

Reply:
To avoid any arguments about this I updated the rules to avoid anything vague - see also https://github.com/astropy/astropy-APEs/pull/61#issuecomment-769284716